client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.html

```diff
@@ -329,3 +333,66 @@
         </div>
     </div>
 </mat-menu>
+
+<!-- Template for dialog for quick editing -->
+<ng-template #motionInfoDialog>
+    <h1 mat-dialog-title>
+        <span>{{ 'Edit details for' | translate }} {{ infoDialog.title }}</span>
+    </h1>
+    <div class="os-form-card-mobile" mat-dialog-content>
+        <mat-form-field>
+            <mat-select placeholder="{{ 'Category' | translate }}" [(ngModel)]="infoDialog.category">
+                <mat-option [value]="null">-</mat-option>
+                <mat-option *ngFor="let category of categories" [value]="category.id">
+                    {{ category.getTitle() | translate }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field>
+            <mat-select placeholder="{{ 'Motion Block' | translate }}" [(ngModel)]="infoDialog.motionBlock">
+                <mat-option [value]="null">-</mat-option>
+                <mat-option *ngFor="let block of motionBlocks" [value]="block.id">
+                    {{ block.getTitle() | translate }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field>
+            <mat-select placeholder="{{ 'State' | translate }}" [(ngModel)]="infoDialog.state">
+                <mat-option *ngFor="let state of infoDialog.nextStates" [value]="state.id">
+                    {{ state.name | translate }}
+                </mat-option>
+                <mat-option *ngFor="let state of infoDialog.previousStates" [value]="state.id">
+                    {{ state.name | translate }}
+                </mat-option>
+                <mat-divider></mat-divider>
+                <mat-option [value]="null">
+                    <mat-icon>replay</mat-icon> {{ 'Reset state' | translate }}
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+        <mat-form-field *ngIf="recommendationEnabled">
+            <mat-select placeholder="{{ recommender }}" [(ngModel)]="infoDialog.recommendation">
+                <mat-option *ngFor="let state of infoDialog.possibleRecommendations" [value]="state.id">
+                    {{ state.name | translate }}
```

Comment:
should be `state.recommendation_label`
(Recommendation is "Acceptance" rather than "accepted")

Reply:
please also consider that there are recommendations with `show_recommendation_extension_field = true` that can be a recommendation combined with a chosable string (see e.g. `motion-repository.service: getExtendedRecommendationLabel` for usage of this)